tests/test_ble_interface_core.py

```
@@ -3459,6 +3571,75 @@ def test_emit_verified_connection_side_effects_sets_reconnected_event(
+            _ClassCoordinator()
+        )
+        assert callable(class_dispatch)
+        cast(Callable[[str], None], class_dispatch)("class-evt")
+        assert class_calls == ["class-evt"]
+        iface.thread_coordinator = _ClassCoordinator()
+        iface._set_thread_event("class-evt-2")
+        assert class_calls == ["class-evt", "class-evt-2"]
+
+        instance_calls: list[str] = []
+        instance_coordinator = SimpleNamespace(
+            _set_event=lambda event_name: instance_calls.append(event_name)
```

Ruff (0.15.6)

[warning] Lambda may be unnecessary; consider inlining inner function

Inline function call

(PLW0108)